modules/dmdIdSystem.md

```diff
@@ -0,0 +1,15 @@
+pbjs.setConfig({
+    userSync: {
+        userIds: [{
+            name: 'dmdId',
+            storage: {
+                name: 'dmd-dgid',
```

Comment:
since you're relying on a value to already exist in this cookie, do you want to be more explicit in the docs (both here and in https://github.com/prebid/prebid.github.io/pull/2909/files) that the `storage.name` must be `'dmd-dgid'`?

Reply:
@smenzer your feedback is implemented. `storage.name` is included in `dmdIdSystem.md` and [https://github.com/prebid/prebid.github.io/pull/2909/files](url). Added more tests to `userId_spec.js`